build.gradle

```
@@ -151,8 +154,18 @@ configure(subprojects - expected_result_generator - tempto_examples) {
     }
 
     signing {
+        def skipSigning = project.findProperty('skipSigning')?.toBoolean() != false
```

Can we leave out the comparison?

```suggestion
        def skipSigning = project.findProperty('skipSigning')?.toBoolean()
```